PyFunceble/cli/__init__.py

```diff
@@ -238,7 +239,7 @@ def tool():  # pragma: no cover pylint: disable=too-many-branches,too-many-state
                     "--complements",
                     action="store_true",
                     help="Switch the value of the generation and test of the complements. "
-                    "A complement is for example `example.org` if `www.example.org` "
+                    "\nA complement is for example `example.org` if `www.example.org` "
```

Comment:
A complement is testing matching domains from the second level from TLD. If you give not.to.be.example.org then example,org would be tested as well and the other way around.

Reply:
It's really just `www.` and without `www.` but that only for a second-level domain (no subdomains)

Reply:
That might be more clear in the text...

"With the complement we explicitly test a domain's usage of the www. and without www. This is limited to the 'www'"

^ This can defiantly be better but it is more clear for a start..

Reply:
It sucks... leave it for brainstorm, ageing to become a more mature textual contents